docs/organizations/default-role.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Reassign the Default role for members
+description: Learn how to assign Clerk's Default role to any other role in an organization.
+content-type: guide
+---
+
+New organization members are intially assigned the [**Default** role](/docs/organizations/roles-permissions#the-default-role-for-members). By default, that role is `org:member`. This role is used as a default in `<OrganizationProfile />` invitations and for organization enrollment with [verified domains](/docs/organizations/verified-domains).
+
+You cannot delete an organization role if it is used as the organization **Default** role. But, you _can_ reassign the **Default** role to any other role.
+
+To reassign the **Default** role:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings/roles).
+1. In the navigation sidebar, select **Organization Settings**.
+1. Select the **Roles** tab. [Create a new role](/docs/organizations/create-roles-permissions#create-a-new-role-for-your-organization) or select an existing role from the list.
+1. At the top right of the screen, open the "..." menu.
+1. From the dropdown, select the **Set as Default role** option.
```

Comment:
Unless i'm missing something, I don't think this is accurate post-redesign! For me, it's:

```suggestion
- Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings/roles).
1. In the navigation bar, select **Configure**.
1. In the sidebar, select **Organization Management** -> **Roles and Permissions**.
1. Select the **Roles** tab.
1. Hover over the role of interest, and click the ... menu on the center-right of the row.
1. From the dropdown, select the **Set as default role** option.
```

That deep-link in step 1 works great for me too.

Reply:
Ah yeah it was a copy paste for the "reassign the creator role" page. I'll update both to be accurate